target/linux/ramips/dts/mt7620a_netgear_ex6100-v1.dts

```diff
@@ -0,0 +1,98 @@
+/dts-v1/;
+
+#include "mt7620a_netgear_ex3700_ex6130.dtsi"
+
+/ {
+	compatible = "netgear,ex6100-v1", "ralink,mt7620a-soc";
+	model = "Netgear EX6100 v1";
+
+	aliases {
+		led-boot = &led_power_amber;
+		led-failsafe = &led_power_amber;
+		led-running = &led_power_green;
+		led-upgrade = &led_power_amber;
+		label-mac-device = &ethernet;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power_green: power_g {
+			label = "ex6100-v1:green:power";
+			gpios = <&gpio2 23 GPIO_ACTIVE_LOW>;
+		};
+
+		led_power_amber: power_a {
+			label = "ex6100-v1:amber:power";
+			gpios = <&gpio2 28 GPIO_ACTIVE_LOW>;
+		};
+
+		router_g {
+			label = "ex6100-v1:green:router";
+			gpios = <&gpio2 25 GPIO_ACTIVE_LOW>;
+		};
+
+		router_r {
+			label = "ex6100-v1:red:router";
+			gpios = <&gpio2 24 GPIO_ACTIVE_LOW>;
+		};
+
+		router_b {
+			label = "ex6100-v1:blue:router-arrow";
```

Comment:
What's "router-arrow"?

Reply:
![IMG_20200222_010100](https://user-images.githubusercontent.com/12028138/75054822-c74f7d80-550e-11ea-8525-e189ddaa4532.jpg)
It's these two "arrow shape" led.

Reply:
![IMG_20200222_015619](https://user-images.githubusercontent.com/12028138/75058766-91ae9280-5516-11ea-8e46-174f10b781f5.jpg)
In user manual it's called Router/Client Arrow.
https://www.netgear.com/support/product/EX6100.aspx#docs

Reply:
Okay, and what does it do?

Reply:
I never seen these LEDs light before. And I can't find any description in user manual. I guess it indicates you should move closer to Router/Client for better signal?

Reply:
Figure 1 tells you the names of the LEDs (if I got it right):

```
		router_link_green {
			label = "ex6100-v1:green:router-link";
			gpios = <&gpio2 25 GPIO_ACTIVE_LOW>;
		};

		router_link_red {
			label = "ex6100-v1:red:router-link";
			gpios = <&gpio2 24 GPIO_ACTIVE_LOW>;
		};

		router_arrow {
			label = "ex6100-v1:blue:router-arrow";
			gpios = <&gpio2 29 GPIO_ACTIVE_LOW>;
		};

		client_link_green {
			label = "ex6100-v1:green:client-link";
			gpios = <&gpio2 20 GPIO_ACTIVE_LOW>;
		};

		client_link_red {
			label = "ex6100-v1:red:client-link";
			gpios = <&gpio2 21 GPIO_ACTIVE_LOW>;
		};

		client_arrow {
			label = "ex6100-v1:blue:client-arrow";
			gpios = <&gpio2 22 GPIO_ACTIVE_LOW>;
		};
```

Despite, is it really "red" or "amber"?

Reply:
> Despite, is it really "red" or "amber"?

I've just read the docs. So, I assume there are green and red LEDs and amber is when both of them are on?

Reply:
> I never seen these LEDs light before. And I can't find any description in user manual. I guess it indicates you should move closer to Router/Client for better signal?

Okay, I also have found nothing. And if you do not have any indications based on what the device does with it runs with vendor OS, I'd just leave them unassigned so the user can do what he/she likes.